Source/Core/DolphinQt/Config/Mapping/MappingWindow.h

```diff
@@ -42,7 +42,9 @@ class MappingWindow final : public QDialog
     // Wii
     MAPPING_WIIMOTE_EMU,
     // Hotkeys
-    MAPPING_HOTKEYS
+    MAPPING_HOTKEYS,
+    // Freelook
+    MAPPING_FREELOOK
```

Comment:
This is really minor, but how about including a comma here anyways? Gets rid of a not-really-a-change (like with `MAPPING_HOTKEYS` above) on the next change.